src/vs/workbench/contrib/search/browser/searchActions.ts

```diff
@@ -589,10 +601,14 @@ export class RemoveAction extends AbstractSearchAndReplaceAction {
 		const opInfo = getElementsToOperateOnInfo(this.viewer, this.element, this.configurationService.getValue<ISearchConfigurationProperties>('search'));
 		const elementsToRemove = opInfo.elements;
 
-		const currentBottomFocusElement = elementsToRemove[elementsToRemove.length - 1];
+		if (elementsToRemove.length === 0) {
+			return Promise.resolve();
```

Comment:
You could make this method `async` so you don't have to return a promise explicitly.